### DISCUSSION TITLE
Changing the language

### DISCUSSION THREAD

**Discussion by ghost** [freeplane/freeplane#1093] [Help]
I want to ask for help. But the language of my freeplane is german. But I have to ask in English. So the german terms of my freeplane version would be not very helpful. So my first question, how do I change the language of the app. I did not find anything in the preference section.

**Comment by EdoFro** [comment]
![image](https://user-images.githubusercontent.com/35700575/231871334-8c59c6db-b922-439e-9964-12cd3d1a90d7.png)
